src/components/navbarbutton.tsx

```diff
@@ -0,0 +1,20 @@
+import React,{Fragment, useState} from 'react'
+import Select, { Option } from 'react-button-navbar';
+ 
+
+
+    function NavbarButton(){
```

Comment:
There are unnecessary spaces on the left of this whole function that can be removed

Reply:
Thank you. Done!